src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -21,7 +22,13 @@
  * The main LogicManager of the app.
  */
 public class LogicManager implements Logic {
-    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
+    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: %s\n"
+            + "Please exit the application by clicking the exit button (i.e., the button marked with an 'X' icon).";
```

Comment:
I still think either you have to tell the user how to recover, or just state the problem only (i.e., without mention exiting at all).